javascript/javascript1/week4/task_1.js

```diff
@@ -0,0 +1,86 @@
+let myName = "";
+let todo = [];
+
+function getReply(command) {
+  const array = command.split(" ");
+  let response="";
```

Comment:
Hi Parul! Nice job with creating a variable for the response! (Or you can also just use return) 😊